packages/@aws-cdk/aws-dynamodb/lib/perms.ts

```diff
@@ -29,3 +29,5 @@ export const READ_STREAM_DATA_ACTIONS = [
   'dynamodb:GetRecords',
   'dynamodb:GetShardIterator',
 ];
+
+export const DESCRIBE_TABLE = 'dynamodb:DescribeTable';
```

Comment:
What do you think of just adding `'dynamodb:DescribeTable'` to the `READ_STREAM_DATA_ACTIONS` array? I think that's basically what we're going for here, and it would save us from doing any other edits to the production code in this PR (OK, minus the comments 😉).

Reply:
Do you mean `READ_DATA_ACTIONS`? `READ_STREAM_DATA_ACTIONS` is for the DynamoDB stream, not reading/writing to the table. 

I did consider adding it to `READ_DATA_ACTIONS` but that had two issues
1. It doesn't cover the write use case, ie `grantWriteData()`
2. If I added it to both `READ_DATA_ACTIONS` and `WRITE_DATA_ACTIONS ` then I'd need code to de-dup it for `grantReadWriteData`

So it seemed simpler to implement it as an additional permission that is added to the necessary grant* calls

Reply:
Yep, I meant `READ_DATA_ACTIONS`.

OK, fair enough!